test/integration/query-interface.test.js

```diff
@@ -190,7 +208,13 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           }
           expect(isAdmin.type).to.equal(assertVal);
           expect(isAdmin.allowNull).to.be.true;
-          expect(isAdmin.defaultValue).to.be.null;
+          switch (dialect) {
+            case 'sqlite':
+              expect(isAdmin.defaultValue).to.be.undefined;
```

Comment:
I think it should, no dialects should report a null default value if there in fact is no default value, null is special.
I'll merge your sqlite fix for now in any case.